Detectors/FIT/FT0/calibration/testWorkflow/FT0CalibCollectorWriterSpec.h

```diff
@@ -61,6 +63,7 @@ class FT0CalibCollectorWriter : public o2::framework::Task
         auto subSpanVect = collectedInfo.subspan(offsetStart, entriesPerChannel[ich]);
         memcpy(&mFT0CalibInfoOut[0], subSpanVect.data(), sizeof(o2::ft0::FT0CalibrationInfoObject) * subSpanVect.size());
         const o2::ft0::FT0CalibrationInfoObject* tmp = subSpanVect.data();
+        LOG(info) << "@@@@@ run ich " << ich << " entries " << entriesPerChannel[ich];
```

Comment:
Is it good to have it as info in the run function? It will spawn a lot of log entries...

Reply:
Yes, you are right. But Its for slewing correction and will not run now. I have to tune slewing calibration - but it will be next task